src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/StacksWorker.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.Diagnostics.Monitoring.UnitTestApp.Scenarios
+{
+    internal class StacksWorker : IDisposable
+    {
+        private EventWaitHandle _eventWaitHandle = new ManualResetEvent(false);
+
+        public class StacksWorkerNested<T>
+        {
+            public void DoWork<U>(U test, WaitHandle handle)
+            {
+                Console.WriteLine(test);
```

Comment:
nit: don't write to stdout; I believe that this stream is used for test execution control; I think stderr is okay to use if you really need to write out something to a std stream